pkg/converters/types/options.go

```diff
@@ -22,25 +22,26 @@ import (
 
 // ConverterOptions ...
 type ConverterOptions struct {
-	Logger           types.Logger
-	Cache            Cache
-	Tracker          Tracker
-	DynamicConfig    *DynamicConfig
-	LocalFSPrefix    string
-	IsExternal       bool
-	MasterSocket     string
-	AdminSocket      string
-	AcmeSocket       string
-	DefaultConfig    func() map[string]string
-	DefaultBackend   string
-	DefaultCrtSecret string
-	FakeCrtFile      CrtFile
-	FakeCAFile       CrtFile
-	AnnotationPrefix []string
-	DisableKeywords  []string
-	AcmeTrackTLSAnn  bool
-	TrackInstances   bool
-	HasGateway       bool
+	Logger                  types.Logger
+	Cache                   Cache
+	Tracker                 Tracker
+	DynamicConfig           *DynamicConfig
+	LocalFSPrefix           string
+	IsExternal              bool
+	MasterSocket            string
+	AdminSocket             string
+	AcmeSocket              string
+	DefaultConfig           func() map[string]string
+	DefaultBackend          string
+	DefaultCrtSecret        string
+	FakeCrtFile             CrtFile
+	FakeCAFile              CrtFile
+	AnnotationPrefix        []string
+	DisableKeywords         []string
+	AcmeTrackTLSAnn         bool
+	TrackInstances          bool
+	HasGateway              bool
+	EnableEndpointSlicesAPI bool
```

Comment:
Use `EnableEPSlices` instead, this helps to avoid changing all the lines of the struct declaration, which in turn helps to avoid conflict when cherry-picking fixes and improvements between branches.